packages/optimizely-sdk/package.json

```
@@ -9,16 +9,19 @@
   "typings": "lib/index.d.ts",
   "scripts": {
     "clean": "rm -rf dist",
-    "test": "mocha ./lib/*.tests.js ./lib/**/*.tests.js ./lib/**/**/*tests.js --recursive --exit --require esm --require lib/tests/exit_on_unhandled_rejection.js",
+    "lint": "eslint 'lib/**/*.js'",
+    "test": "mocha --exit -r esm -r lib/tests/exit_on_unhandled_rejection.js 'lib/**/*tests.js'",
```

After running this locally, I noticed this glob pattern picks up a file that it shouldn't: `lib/index.browser.umdtests.js`. That one is intended to be run via Karma (`npm run test-umdbrowser`) and doesn't work with a plain mocha run. Can we use this pattern instead? 
```suggestion
    "test": "mocha --exit -r esm -r lib/tests/exit_on_unhandled_rejection.js 'lib/**/*.tests.js'",
```

---

Figures... that was the pattern I had before, until I noticed that the original code had this:

`mocha ./lib/*.tests.js ./lib/**/*.tests.js ./lib/**/**/*tests.js`

(The last pattern was `*tests.js`.)

If I could be so bold, I would suggest this indicates a problem with the way the tests are organized. If the umdtests file shouldn't be run as part of the mocha suite, shouldn't it be placed elsewhere? (Like `test/umd-integration.js` or something?)

---

I've updated the PR by removing that last commit.

---

Yea that's fair, we should move it elsewhere. I'll follow up on this.

---

Regardless, the "restored" glob is back to excluding umdtests, so that extraction need not affect this PR :D